Automattic/wp-calypso
# Onboarding: Adjust styles according to design feedback

#### Changes proposed in this Pull Request

As design feedback EuqfSnWpfYx8fgiBlVmbuA-fi-2875%3A1519, adjust styles as followed:
* Replace illustration image and add margin-top to the content for both of intent and give your blog a name
* Intent
  * styles of button
  * opacity of separating line
* Give your blog a name
  * styles of the label, input, and tips
  * styles of button
* Design preview
  * margin-top of title

#### Testing instructions



* Go to `/start/setup-site?siteSlug=<your_site>&flags=signup/hero-flow`
* Check the design in each page 



Related to https://github.com/Automattic/wp-calypso/issues/55603, https://github.com/Automattic/wp-calypso/issues/56957
